File: Com4send.jl
using LibSerialPort

function WriteRS232C(txt, portname="COM11", baudrate=115200)
    LibSerialPort.open(portname, baudrate) do sp
        sleep(0.5)
        write(sp, txt * "\r\n")
    end
end

function ReadRS232C(portname="COM11", baudrate=115200)
    txt = ""
    LibSerialPort.open(portname, baudrate) do sp
        sleep(0.5)
        if bytesavailable(sp) > 0
            txt = String(read(sp))
        end
    end
    return txt
end


portname = "COM11"
baudrate = 115200

WriteRS232C("TESTWrite", portname, baudrate)

File: Com4send.py
import serial

# readSer = serial.Serial('Com4', 115200)

portname = "COM4"
baudrate = 115200


readSer = serial.Serial(portname, baudrate, timeout=1)
# c = readSer.read()  # 1 byte
# string = readSer.read(10)  # 10 byte
# line = readSer.readline()  # 1 line (upto '\n')
# print("Read Serial:")
# print(c)
# print(string)
# print(line)
# readSer.close()
serialCommand = "test\r\n"
readSer.write(serialCommand.encode())
readSer.close()
